### ISSUE TITLE
[Feature Request] Calendar module: Hide recurringCountTitleForSpecificEvents

### ISSUE THREAD

**Issue by jupadin** [MichMich/MagicMirror#2700]
Hi,

**TL;DR**
Given a calendar with birthday entries, not knowing the date of birth for each entry, I want to hide the recurring count title.
*In case you don't like it due to the given reasons, just close this issue and related PR.*

According to post in https://github.com/MichMich/MagicMirror/issues/2639#issuecomment-909013378, I thought about a solution to this and came up with the following:

**Use case**:
You want to have a birthday calendar, based on the default calendar module, because it is such a great module and has a lot of functionality \o/.
**Problem:**
I do not know the year of birth for all of my entries, which would result in having a recurring count title like seen here (knowing the year of birth for the event _My birthday_ and not knowing the year of birth for the event _Another birthday_ - just chose a random year since I have to specify one in the ics file...):
![Bildschirmfoto 2021-10-22 um 10 57 10](https://user-images.githubusercontent.com/84035374/138427459-6536a290-c2d0-498d-ab2a-46d1094e6748.png)

**Solution:**
In the config, you can specify a year (under the given option: `hideRepeatingCountTitleOnYear`, which default value is `false`, such that it does not harm anything on default), for which the recurringCountTitle will be hidden.
If this config parameter is set, the situation is as follows (since I specified that the recurring count title for all events having a start year of `1920` should be hidden):
![Bildschirmfoto 2021-10-22 um 10 57 44](https://user-images.githubusercontent.com/84035374/138427673-6d2ea724-6e6f-411e-b37a-6c0b55f8eb92.png)

**Advantage:**
I can specify all my events with a given year of birth (or call it starting year), for which the recurring event title is hidden and I don't have to use a third party module, do some hacky things in the configuration for each individual calendar url or have to use two separate calendar ics-files (one for knowing the year of birth and one for the other case) - with btw. would lead to copying one event to the other file when knowing the year of birth.

I would appreciate your thoughts and ideas on this :-)
